spec/02-integration/04-admin_api/25-add_delete_workspaces_spec.lua

```diff
@@ -0,0 +1,50 @@
+local helpers = require "spec.helpers"
```

Comment:
Nitpick: maybe name this file accordingly to how others are neme, after the main entity? eg, `spec/02-integration/04-admin_api/25-workspaces_spec.lua`